docs/package.json

```
@@ -10,7 +10,7 @@
     "start": "yarn pre-build && yarn develop",
     "clear": "docusaurus clear",
     "develop": "docusaurus start --port 3000",
-    "build": "docusaurus build --out-dir build/docs/rasa",
+    "build": "`which node` --max-old-space-size=8192 `which npx` docusaurus build --out-dir build/docs/rasa",
```

isn't the following shortest?
```suggestion
    "build": "NODE_OPTIONS='--max-old-space-size=8192' docusaurus build --out-dir build/docs/rasa",
```